.env.prod

```diff
@@ -3,7 +3,9 @@ REACT_APP_ENVIRONMENT=staging
 REACT_APP_ETHEREUM_RPC_URL=https://mainnet.eth.aragon.network/
 REACT_APP_SNAPSHOT_URL=https://hub.snapshot.page/api
 REACT_APP_TOKENSETS_API=https://api.tokensets.com/public
```

Comment:
i guess we can deprecate this env in favor of `REACT_APP_TOKENSETS_URL`?

Reply:
that's the plan, unfortunately the `/public` makes it hard to reuse